### ISSUE TITLE
Proximity sensor problem with locked screen

### ISSUE THREAD

**Issue by GI9VANNI** [ubports/ubuntu-touch#1639]
### Description of the feature
The proximity sensor does not respond correctly if I turn off the screen with the side button during a call.

### Illustrations
- Device:    Nexus 5
- Channel:  Stable
- Build:       OTA-15

### Steps to reproduce
If I turn the screen off during a call with the side button, the screen turns back on if I pass something near the proximity sensor.
Instead, the screen should remain off.

### Expected behavior
The screen should always remain off if I press the side button to turn it off.

### Actual behavior
If I turn the screen off during a call with the side button, the screen turns back on if I pass something near the proximity sensor.

**Comment by Flohack74** [comment]
Ok I basically understand you. But this is how you can work around for the moment:
- Just put the phone in your pocket. Proximity sensor will detect your pocket and even with notifications incoming, it wont turn on.
Can you try if this works without problems ;)

**Comment by UniversalSuperBox** [comment]
Indeed, it seems like Android phones will keep the screen off if you turn it off manually. Ubuntu touch will wake the screen up again when the proximity sensor transitions back to "far"
